presto-product-tests/bin/run_on_docker.sh

```diff
@@ -24,6 +24,10 @@ function hadoop_master_container(){
   environment_compose ps -q hadoop-master
 }
 
+function presto_master_container(){
```

Comment:
This function will be unused after addressing the comments and should be then deleted